invenio_search/ext.py

```diff
@@ -405,6 +405,20 @@ def _build(tree_or_filename, alias=None):
                     name=action["alias"],
                     ignore=ignore,
                 )
+    def update_mapping(self, index):
+        """Update mapping of the existing index."""
+        mapping_path = self.mappings[index]
+        final_index = build_alias_name(index)
+
+        index_ = self.client.indices.get(final_index)
+        full_index_name = list(index_.keys())[0]
+        index_ = dsl.Index(full_index_name, using=self.client)
```

Comment:
not a 100% sure, managed to track till [here](https://github.com/opensearch-project/opensearch-py/blob/23611ebda486c8651d36e993335652f4c6b4f87e/opensearchpy/client/cat.py#L207). Would the `values` in line 414 suffice?

```python
index_ = self.client.indices.get(final_index).values()[0]
```

Reply:
unfortunately self.client.indices.get returns a dict type and I need a full object to be able to access the .put_mappings method